pivy-importer/src/main/groovy/com/linkedin/python/importer/ivy/IvyFileWriter.groovy

```diff
@@ -85,6 +79,44 @@ class IvyFileWriter {
 
         def ivyText = writer.toString()
 
-        new File(destDir, "${name}-${version}.ivy").text = ivyText
+        if (packageType == SdistDownloader.SOURCE_DIST_PACKAGE_TYPE) {
+            new File(destDir, "${name}-${version}.ivy").text = ivyText
+        } else if (packageType == WheelsDownloader.BINARY_DIST_PACKAGE_TYPE) {
+            new File(destDir, "${name}-${version}-${classifier}.ivy").text = ivyText
+        }
+    }
+
+    private getPublicationsMap() {
+        def publicationMap = archives.collect { artifact ->
+            def ext = artifact.filename.contains(".tar.") ? artifact.filename.find('tar\\..*') : FilenameUtils.getExtension(artifact.filename)
+            String filename = artifact.filename - ("." + ext)
+            def source = 'sdist' == artifact.packageType
+            def map = [name: name, ext: ext, conf: source ? 'source' : 'default', type: ext == 'whl' ? 'zip' : ext]
```

Comment:
Seems like if ext == 'whl', then the type will be 'zip'? How does that work?

Reply:
This change made by Linkedin, I prefer to use "whl" as well, they use "tar.gz" as type for sdist, I think it should be "source" or "sdist" instead. 
see http://ant.apache.org/ivy/history/2.3.0/ivyfile/artifact.html
The official explanation here is a little confused, it mentioned "the type of the published artifact. It's usually its extension, but not necessarily.", but in the example, 
<artifact name="foo-src" type="source" ext="zip" conf="src" />
I guess both are fine, so I just follow whatever LinkedIn is using for now, but we can change it if we decide to.

Reply:
Ok. It just seemed weird, but if it works, let's keep it this way

Reply:
I have been thinking about it. I think maybe we should make it as "whl" which is a better type, I don't think they used this type actual since they only get sdist packages in the past. And this is the only diff from what we have, we shall help them to do the right thing.